src/components/SpreadsheetUploader/SpreadsheetUploader.module.scss

```diff
@@ -20,8 +20,11 @@
   }
   &_templatelink {
     color: $primary-blue;
+    display: block;
     text-align: center;
+    font-size: 13px;
```

Comment:
Can we use `rem` instead of `px` so that the font size will scale properly if the user sets their font-size differently? That's pretty important for accessibility.